### ISSUE TITLE
Version mismatch when added to Jekyll

### ISSUE THREAD

**Issue by sajeshcherian** [aarongustafson/jekyll-webmention_io#146]
Getting this error about version mismatch when trying to add the ruby gem to Jekyll

```
Bundler could not find compatible versions for gem "jekyll":
  In Gemfile:
    jekyll (~> 4.1)

    jekyll-webmention_io was resolved to 2.0.1, which depends on
      jekyll (< 4.0, >= 3.0)
```

**Comment by aarongustafson** [comment]
I kept having Ruby/Jekyll issues when upgrading my Mac, so I have moved off of Jekyll and will not be working on this project anymore, going forward. I am going to flag this as won’t fix, but leave it open in case someone else wants to pick up the project from here.

**Comment by fancypantalons** [comment]
This has been resolved in the master code-base, but obviously it won't be completely resolved until a new package is published to rubygems.

Until then, feel free to reference the github repository in your gemfile and things should work.

**Comment by fancypantalons** [comment]
I believe this is now resolved in 3.3.6 which has been pushed to RubyGems.
